docs/content/concepts/metadata-tags/asset-metadata.mdx

```diff
@@ -129,9 +129,9 @@ def topstories(context: AssetExecutionContext) -> MaterializeResult:
     )
 ```
 
-### Column-level lineage
+### Table metadata
```

Comment:
Hmm, I wasn't sure how to properly categorize it, since column lineage is a kind of table metadata. Added a link here, but kept the header as table metadata. Let me know what you think.